src/azure-cli/azure/cli/command_modules/sql/tests/latest/recordings/test_sql_vnet_mgmt.yaml

```diff
@@ -1304,7 +1304,7 @@ interactions:
       - python/3.5.3 (Windows-2012ServerR2-6.3.9600-SP0) msrest/0.6.11 msrest_azure/0.6.2
         azure-mgmt-network/9.0.0 Azure-SDK-For-Python AZURECLI/2.1.0
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Network/locations/eastus/operations/757ec43b-e47a-4506-9100-4d13a33436cb?api-version=2019-11-01
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Network/locations/eastus/operations/757ec43b-e47a-4506-9100-4d13a33436cb?api-version=2020-03-01
```

Comment:
It's cheating! 😆

Reply:
```diff
- {Network}Bump network api version to 2020-03-01
+ {Network} Bump network api version to 2020-03-01
```

Reply:
cannot run about 10~20 tests. I manually checked the logic and they are just using the simplest commands in Network.

Reply:
You are so cute when you are being serious. 🤗